From fed03275e6148fcbb9e2bf0f0eedd2d629dfe767 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Wed, 4 Oct 2017 18:24:04 +0000
Subject: [PATCH] Use standard autoconf python check

Clear Linux only ships python 2.7+ so do not use the manual test.
---
 configure.ac | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/configure.ac b/configure.ac
index 200a7cc..8ada462 100644
--- a/configure.ac
+++ b/configure.ac
@@ -31,12 +31,7 @@ AM_CONDITIONAL([HAVE_PYTHON], test -n "$PYTHON")
     [echo "Then run configure again before attempting to build Satyr."]
 [fi]
 
-# Just PKG_CHECK_MODULES([PYTHON], [python]) works only with python2.7+
-# Below, if python is not found, we set up for python2.6 w/o checking:
-PKG_CHECK_MODULES([PYTHON], [python2],,[
-    PYTHON_LIBS='-L/usr/lib64 -lpython2.6'
-    PYTHON_CFLAGS='-I/usr/include/python2.6'
-])
+PKG_CHECK_MODULES([PYTHON], [python])
 
 AC_ARG_ENABLE([python-manpage],
               [AS_HELP_STRING([--disable-python-manpage],
-- 
2.14.2

